<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateShipmentsWindow</class>
 <widget class="QDialog" name="CreateShipmentsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>437</width>
    <height>522</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create Shipment</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create Shipment</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Regions</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>From:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Supply Type</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="srcRegionSupplies">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>331</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Quantity</string>
   </property>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999999</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>178</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateShipmentsWindow</receiver>
   <slot>CreateShipmentsWindowSubmitButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CreateShipmentsWindow</receiver>
   <slot>CreateShipmentsWindowCancelButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CreateShipmentsWindow</receiver>
   <slot>notifyUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_3</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CreateShipmentsWindow</receiver>
   <slot>notifyUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CreateShipmentsWindowSubmitButtonHandler()</slot>
  <slot>CreateShipmentsWindowCancelButtonHandler()</slot>
  <slot>notifyUser()</slot>
 </slots>
</ui>
